src/Features/LanguageServer/Protocol/Handler/AbstractRequestHandler.cs

```diff
@@ -18,6 +19,7 @@ protected AbstractRequestHandler(ILspSolutionProvider solutionProvider)
             SolutionProvider = solutionProvider;
         }
 
+        public virtual TextDocumentIdentifier? GetTextDocumentIdentifier(RequestType request) => null;
```

Comment:
A follow up PR will make this abstract (actually removing this class altogether)